### PR TITLE
Update existing SignalR Docs to RC1

### PR DESCRIPTION
**Pull request by anurse**
* Updates the JavaScript code to use HubConnectionBuilder (since the API from Preview 2 no longer works)
* Sample clean-up:
  * Use consistent HTML/JS
  * Remove broken scripts to reduces Console spew (they weren't being used anyway)
  * Add `package.json` and install `@aspnet/signalr` in each project.
  * Add MSBuild goop to projects to copy from `node_modules` into the `wwwroot`, so the sample can be built and run locally using only `npm install` followed by `dotnet run`.

We'd like to get this in fairly soon since RC1 has been released and the existing samples don't work with RC1.

Preview URLs:
* [Getting Started](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/get-started?view=aspnetcore-2.1&branch=pr-en-us-6249&tabs=visual-studio)
* [Hubs](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/hubs?view=aspnetcore-2.1&branch=pr-en-us-6249)
* [JavaScript Client](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/javascript-client?view=aspnetcore-2.1&branch=pr-en-us-6249)

### CODE REVIEW

File: aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js [hunk 1]
```diff
@@ -18,11 +13,11 @@ document.getElementById("sendButton").addEventListener("click", function (event)
     var user = document.getElementById("userInput").value;
     var message = document.getElementById("messageInput").value;
     connection.invoke("SendMessage", user, message).catch(function (err) {
-        return console.error;
+        return console.error(err.toString());
```
Comment (BrennanConroy) [inline aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js]: nit: `err => console.error(err.toString())`

**Comment by Rick-Anderson** [comment]
@anurse Might be best to merge it now. It's much better than the old version for sure. Let me know and I'll merge it to live.

File: aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js]: @anurse  fix that and I'll get it merged to live.

**Comment by BrennanConroy** [comment]
https://github.com/aspnet/Docs/blob/253210f522739485b730b7e892b909a0f29eb307/aspnetcore/signalr/get-started/sample/Startup.cs#L33
and
https://github.com/aspnet/Docs/blob/084cdbf3f05e9dec20719863cad3a88925dd29d4/aspnetcore/signalr/hubs/sample/Startup.cs#L33
Should have the AllowCredentials setting

File: aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js (hunk 1 above)
Comment (anurse) [reply aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js]: This is the ES5 transpilation, it doesn't use arrow functions. It's generated by Babel

**Comment by anurse** [comment]
@Rick-Anderson With the latest commit to update the Startup.cs, I think we're good to merge this live. We can react to further changes as they arise.

**Pull request merged**